### PR TITLE
Feature/cadence deps commodities

### PR DESCRIPTION
**Pull request by giuseppe-bertucci**
* **What kind of change does this PR introduce (Bug fix, feature, docs update, ...)?**
  The PR introduces as new commodities cadence orchestrator and its dependancies (cassandra, cli) 

* **What is the current behavior?**
no cadence/cassandra available 

* **What is the new behavior (if this is a feature change)?**
cadence included and ready to use

* **Does this PR introduce a breaking change? If so, what actions will users need to take in order to regain compatibility?**
no breaking changes

### All submissions

* [X] Have you followed the guidelines in our [Contributing](CONTRIBUTING.md) document?
* [X] Is this PR targeting the `develop` branch, and the branch rebased with commits squashed if needed?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### New feature and bug fix submissions

* [X] Has your submission been tested locally?
* [] Has documentation such as the [README](/README.md) been updated if necessary?
(documentation not updated yet, PL is mainly meant to collect a feedback for the moment. Docs activity is expected in the next couple of days)

### CODE REVIEW

**Review by sichapman — CHANGES_REQUESTED** [review]
Can the README be updated to add the new commodities to the list of available ones (https://github.com/LandRegistry/common-dev-env#configurationyml), and also a small section detailing the exposed ports (e.g. like https://github.com/LandRegistry/common-dev-env#rabbitmq)

File: scripts/docker/cadence/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+FROM ubercadence/server:master-auto-setup
```
Comment (sichapman) [inline scripts/docker/cadence/Dockerfile:2]: Is it OK to tie this to what I assume is the constantly-under-development master branch, as opposed to a specific release version?

File: scripts/docker/cadence_cli/Dockerfile [hunk 2]
```diff
@@ -0,0 +1 @@
+FROM ubercadence/cli:master
```
Comment (sichapman) [inline scripts/docker/cadence_cli/Dockerfile:1]: Should the cadence and cadence_cli commodities be merged? They can remain separate images/containers, just always activated together (think how "auth" commodity spins up both keycloak and openldap)

File: scripts/docker/cadence/Dockerfile (hunk 1 above)
Comment (andrew-bartlett-gh) [reply scripts/docker/cadence/Dockerfile:2]: We should be using ubercadence/server:0.14.0

File: scripts/docker/cadence_cli/Dockerfile (hunk 2 above)
Comment (giuseppe-bertucci) [reply scripts/docker/cadence_cli/Dockerfile:1]: we're considering to replace it with an alias in `add-aliases.sh`, would it work better? 

```
alias cadence-cli='docker run --network=host --rm ubercadence/cli:0.8.2`
```

Comment (andrew-bartlett-gh) [reply scripts/docker/cadence_cli/Dockerfile:1]: that looks good to me

File: scripts/docker/cadence/Dockerfile (hunk 1 above)
Comment (giuseppe-bertucci) [reply scripts/docker/cadence/Dockerfile]: update to 0.14.1 (latest at time of writing)

File: scripts/docker/cadence_cli/Dockerfile (hunk 2 above)
Comment (giuseppe-bertucci) [reply scripts/docker/cadence_cli/Dockerfile]: after testing the 0.8.2 was not working properly, we had to switch to 0.7.0. resulting alias is 
```
alias cadence-cli="docker run --rm ubercadence/cli:0.7.0 --address host.docker.internal:7933"
```

**Pull request merged**